source/atlas-cli-deploy-docker.txt

```diff
@@ -0,0 +1,148 @@
+.. _atlas-cli-deploy-docker:
+
+=========================================
+Create a Local Atlas Deployment in Docker
+=========================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This tutorial shows you how to use the ``atlas deployments`` command to 
```

Comment:
Suggestion: link to the command's page here so they can see all the options for the command
```suggestion
This tutorial shows you how to use the :ref:`atlas-deployments` command to 
```